qemu/tests/cfg/win_virtio_single_driver.cfg

```diff
@@ -0,0 +1,50 @@
+-win_virtio_single_driver:
+    type = win_virtio_single_driver
```

Comment:
backup_image_before_test and kill_after_test should be yes. so that we can ensure this case will not impact next when test failed.

Thanks,
Xu